pom.xml

```diff
@@ -383,10 +383,11 @@
             <version>${org.ops4j.pax.exam.version}</version>
             <scope>test</scope>
         </dependency>
+        <!-- can't go beyond org.awaitility 3.1.6 because PAX exam 4.0.0 is hard coded to use this-->
```

Comment:
I'm not really sure why o.a.sling.testing.paxexam has changed the awaitility version to the older 3.x version instead of the 4.x version that it used to be.

I think we should try to not go backwards to the 3.x version of awaitility if we can avoid it.  I think the syntax of java.time.Duration is more readable than the older syntax of org.awaitility.Duration constructors.

I think that a workaround could be accomplished by providing a custom ContentloaderTestSupport#awaitility method to provide a different implementation that pulls in the latest awaitility library instead of the old one.

Something like this new method in ContentloaderTestSupport should work?
```
    /**
     * Replacement for {@link SlingOptions#awaitility()} to utilize a newer version of awaitility
     *
     * NOTE: may remove this at a later date and go back to {@link SlingOptions#awaitility()} whenever
     * {@link SlingVersionResolver} provides these versions or later
     */
    public static ModifiableCompositeOption awaitility() {
        return composite(
            mavenBundle().groupId("org.awaitility").artifactId("awaitility").versionAsInProject(),
            mavenBundle().groupId("org.hamcrest").artifactId("hamcrest").version(SlingOptions.versionResolver)
        );
    }

```